nautobot/extras/api/views.py

```diff
@@ -283,11 +284,16 @@ def run(self, request, class_path):
         input_serializer = serializers.JobInputSerializer(data=request.data)
         input_serializer.is_valid(raise_exception=True)
 
-        data = input_serializer.data["data"]
+        data = input_serializer.data["data"] or {}
```

Comment:
Why not `input_serializer.data.get("data", {})`?

Reply:
As written this also handles the case where `input_serializer.data["data"]` is present but set to an empty string or `None`.

Reply:
The use of `or` in this fashion is a somewhat weird habit I can’t seem to shake. I totally understand if we prefer a more explicit method to make the intent clearer.